praveglet/README.md

```diff
@@ -0,0 +1,47 @@
+
+# Praveglet
+
+Praveglet is an app that can be used to interact with Pravega. This utility can perform the following operations
+* Create a Scope
+* Create a Stream
+* Write events into a Stream from STDIN or FIFO.
+* List Scopes
+* List Streams
+* List Streams for the provided Tag.
+
+### Usage documentation:
+```
+Praveglet 0.0.1
+Utility to interact with Pravega
+
+USAGE:
+praveglet [FLAGS] [OPTIONS] <SUBCOMMAND>
+
+FLAGS:
+-a, --enable-auth    Enable authorization, default is false
+-h, --help           Prints help information
+-V, --version        Prints version information
+
+OPTIONS:
+-u, --controller-uri <controller-uri>  To enable TLS use uri of the format
+tls://ip:port [default: tcp://127.0.0.1:9090]
+
+SUBCOMMANDS:
```

Comment:
It doesn't have to be in this PR, but it would also be good to have support for:
* Writing to a byteStream
* Reading from a byteStream.
* Reading from a normal stream (Though this is somewhat messy because we will need to first create a reader group)